mod/starred.php

```diff
@@ -39,7 +43,7 @@ function starred_init(&$a) {
 		if(strpos($return_path, '?')) $rand = "&$rand";
 		else $rand = "?$rand";
```

Comment:
Standards: Could you please review this whole conditional statement for brackets and spaces? 😊